libs/langgraph/src/pregel/retry.ts

```diff
@@ -167,6 +167,16 @@ async function _runWithRetry(
           2
         )} seconds (attempt ${attempts}) after ${errorName}: ${error}`
       );
+      // Clear checkpoint_ns seen (for subgraph detection)
+      const checkpointNs = pregelTask.config?.configurable?.checkpoint_ns;
+      if (checkpointNs) {
+        _SEEN_CHECKPOINT_NS.delete(checkpointNs);
```

Comment:
does this raise an error if not found?

Reply:
No, should be idempotent:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/delete